src/makefile

```
@@ -1,8 +1,14 @@
-libAzStorage.so: AzStorage.o
+local:
+	$(CC) `curl-config --cflags` -O3 -fopenmp -fPIC -c -o AzStorage.o AzStorage.c
+	$(CC) -shared -fopenmp -o libAzStorage.so AzStorage.o `curl-config --libs`
+
+yggdrasil: libAzStorage.${dlext}
+
+libAzStorage.${dlext}: AzStorage.o
 	$(CC) -shared -fopenmp -o $@ $^ `curl-config --libs`
 
-AzStorage.o: AzStorage.c
+AzStorage.$(dlext): AzStorage.c
 	$(CC) `curl-config --cflags` -O3 -fopenmp -fPIC -c -o $@ $^
 
 clean:
-	rm -rf *.so *.o
+	rm -rf *.$(dlext) *.o
```

Maybe you want to keep `*.so`
```suggestion
	rm -rf *.so *.$(dlext) *.o
```